demo/index.html

```diff
@@ -21,15 +21,30 @@
       <br/>
       <tt>myForm.$error.dimensions (height < 400) = {{!!myForm.$error.dimensions}}</tt>
       <br/>
+      <tt>myForm.$error.size (size < 2097152) = {{!!myForm.$error.size}}</tt>
+      <br/>
     </p>
     <p>
-      <input type="file" name="myImage" image-with-preview ng-model="image" accept="image/jpeg,image/png" dimensions="height < 400">
+      <input type="file"
+            name="myImage"
+            image-with-preview
+            ng-model="image"
+            accept="image/jpeg,image/png"
+            dimensions="height < 400"
+            size="size < 2097152">
```

Comment:
Because the value is unit-less, I think it would be more user friendly if it would read as `size="sizeInBytes < 2097152"`. On the other hand the dimension values also lack units (could be `vh` instead of `px`, for example) and the unit used here is the same used by the `size` attribute of JS `File`, so maybe it's not too much of a problem .

Reply:
Agreed, though I liked the simplicity of `size`. We can specify in the readme that it's in pure bytes.